lib/recurly/hosted-fields.js

```diff
@@ -29,13 +30,55 @@ export class HostedFields extends Emitter {
     this.errors = [];
     this.initQueue = [];
     this.readyState = 0;
+    this.onTabNext = this.onTabNext.bind(this)
+    this.onTabPrevious = this.onTabPrevious.bind(this)
     this.configure(options);
     this.inject();
     this.on('hostedField:state:change', this.update.bind(this));
     this.on('bus:added', bus => {
       this.bus = bus;
       this.fields.forEach(hf => bus.add(hf));
     });
+
+    this.on('hostedField:tab:previous',this.onTabPrevious)
+    this.on('hostedField:tab:next',this.onTabNext)
+  }
+
+  onTabNext(body) {
+    let type = body.type
+    let field = this.fields.find(function(f){ return f.type === type })
+    let allFields = document.querySelectorAll("[data-recurly]")
```

Comment:
Outside of using an attribute like tabindex I'm not sure the best way to find what the next or previous tabable element should be. I'd need to reimplement the rules for which elements are tabbable to get the right set, and then what order the iframes fit in. I started down that road a little but it seemed a bit ridiculous for me. I thought it would be better to have a way for support to tell people how to make sure tabbing works (hey, make sure you have [recurly-data] on your recurly hosted field divs and at least the immediate elements before and after.)